### ISSUE TITLE
loss compute

### ISSUE THREAD

**Issue by JiangWeixian** [wiseodd/generative-models#18]
i think, should use `nn.BCEloss`, not `-(torch.mean(torch.log(D_real)) + torch.mean(torch.log(1 - D_fake)))`
